code/modules/ninja/suit/ninjaDrainAct.dm

```diff
@@ -261,4 +261,19 @@ They *could* go in their appropriate files, but this is supposed to be modular
 		spark_system.set_up(5, 0, loc)
 		playsound(src, "sparks", 50, 1)
 		visible_message("<span class='danger'>[H] electrocutes [src] with [H.p_their()] touch!</span>", "<span class='userdanger'>[H] electrocutes you with [H.p_their()] touch!</span>")
-		electrocute_act(25, H)
+		electrocute_act(15, H)
+
+		src.Knockdown(G.stunforce)
```

Comment:
This. This is not the proper use of src at all.

Reply:
All righto. Fixed.
tbh, it came from me straight copying stunbaton code
`
L.Knockdown(stunforce)
L.adjustStaminaLoss(stunforce*0.1, affected_zone = (istype(user) ? user.zone_selected : BODY_ZONE_CHEST))//CIT CHANGE - makes stunbatons deal extra staminaloss. Todo: make this also deal pain when pain gets implemented.
L.apply_effect(EFFECT_STUTTER, stunforce)
`